components/Fields/Field.vue

```diff
@@ -222,15 +227,7 @@ export default defineNuxtComponent({
         <Phone :number="phone" />
       </div>
 
-      <div
-        v-for="item in properties[field.field]"
-        v-else-if="field.field === 'website'"
-        :key="`website_${item}`"
-      >
-        <ExternalLink :href="item" target="_blank">
-          {{ item }}
-        </ExternalLink>
-      </div>
+      <Website v-else-if="isWebsite" :urls="properties[field.field]" />
```

Comment:
Please, keep the loop outside the corposant, like the others.

Reply:
It not really recommended to use v-for and v-if on the same element:
- https://vuejs.org/guide/essentials/conditional.html#v-if-with-v-for
- https://vuejs.org/style-guide/rules-essential.html#avoid-v-if-with-v-for

Why is this important ?

In my opinion it make sense that Website has a loop because he have an Array prop (urls).
If we keep the loop outside then the Website component is useless has it become the same as ExternalLink.

Reply:
It is not about where the loop should be. We can make refactoring to change it. But here is to keep homogeneous with the other fields.

Reply:
So I'm keeping Website component even if it's useless to have one ?

Reply:
Right, download and email are also just external link. As you wish, you can use it or remove it.